frontend/src/shared/styles/tokens.css

```
@@ -0,0 +1,107 @@
+  --info-ink: oklch(0.38 0.13 250);
+
+  /* dark variant */
+  --dark-bg: oklch(0.155 0.012 260);
+  --dark-bg-2: oklch(0.195 0.012 260);
+  --dark-line: oklch(0.30 0.012 260);
+  --dark-ink: oklch(0.96 0.005 100);
+  --dark-ink-2: oklch(0.78 0.010 260);
+  --dark-ink-3: oklch(0.58 0.012 260);
+
+  /* type with Korean fallback */
+  --sans: 'Inter Tight', 'Pretendard Variable', -apple-system, BlinkMacSystemFont, system-ui, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)